src/main/java/com/avengers/gamera/entity/User.java

```diff
@@ -31,7 +30,7 @@ public class User {
     @Column(unique = true, nullable = false)
     private String email;
 
-    @Column(nullable = false)
+    @Column(columnDefinition="char(64)")
```

Comment:
We don't use DDL to change table structure or definition, so this will not work.

Reply:
For test use, forgot to remove 😂